Road/notebooks/feature_selected.ipynb

```diff
@@ -0,0 +1,1189 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    data['reply_to'] = data['reply_to'].apply(lambda x: FS.extract_id(json_string=x, target_id='reply_to_msg_id'))
</pre><p>Same comment regarding line length in PEP8 convention.</p><p>You can do sth like this:</p><p><code>data['reply_to'] = data['reply_to'].apply(</code></p><p><code>&nbsp;&nbsp;lambda x: FS.extract_id(json_string=x, target_id='reply_to_msg_id')</code></p><p><code>)</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Road-Status/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>